src/modules/categories/components/category-list/category-list.jsx

```diff
@@ -19,11 +19,11 @@ const CategoryList = (p) => {
         return (
           <div
             className={categoryStyling}
-            key={category !== null ? JSON.stringify(category) : `${JSON.stringify(category)}${categoryIndex}`}
+            key={category ? JSON.stringify(category) : `${JSON.stringify(category)}${categoryIndex}`}
           >
             <Category
-              category={category !== null ? category.category : 'null-category'}
-              popularity={category !== null ? category.popularity : 0}
+              category={(category && !!category.category) ? category.category : 'null-category'}
```

Comment:
Isn't this already handled in the selector?

Reply:
yeah, just making absolutely sure. Bunk/bad categories can still be in the state, they shouldn't be, but can be. the selector should be filtering them out before getting here, but in the off chance that somehow it didn't for whatever reason, i wanted to be explicit.